### PR TITLE
Create the UI

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Create the UI as required through the main and app file

### CODE REVIEW

File: classes/student.rb [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-# student subclass
 
 class Student < Person
-  def initialize(classroom, _id, _age, _name = 'unkown', parent_permission: true)
-    super
-    @classroom = classroom
+  attr_accessor :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```
Comment (ahmetbozaci) [inline classes/student.rb]: - [x] Please fix the error when creating a student.
image

_Next code reviewer please check if creating rentals or list the rentals. Right now ı can't create any person so ı can't create the rentals._

File: classes/teacher.rb [hunk 2]
```diff
@@ -1,9 +1,8 @@
 require_relative 'person'
-# teacher subclass
 
 class Teacher < Person
-  def initialize(specialization, _id, _age, _name = 'unkown', parent_permission: true)
-    super
+  def initialize(specialization)
+    super()
```
Comment (ahmetbozaci) [inline classes/teacher.rb]: - [x] Please fix the error when creating a person.

image

_Next code reviewer please check if creating rentals or list the rentals. Right now ı can't create any person so ı can't create the rentals._

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @eng-mohamed-eldeeb  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] It will be good if you move main.rb file outside the classes folder. So we can just run with type `ruby main.rb`

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @eng-mohamed-eldeeb , 

Great work on making the changes requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**